docs/upgrading.md

```diff
@@ -65,7 +88,7 @@ Note: the provided [migration script](../files/migrate-v090-to-v200.sh) can be u
 helm --tiller-namespace giantswarm delete --purge cert-manager
```

Comment:
Does this still apply? This works with helm 3 and without tiller, right?

Reply:
This is not scope of this PR.. let’s ignore this